tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -210,9 +210,27 @@ static DetachedSpan detachInternal(String operation, SpanType type) {
                 ? maybeCurrentTrace.getTraceId() : Tracers.randomId();
         boolean sampled = maybeCurrentTrace != null
                 ? maybeCurrentTrace.isObservable() : sampler.sample();
+        Optional<String> parentSpan = getParentSpanId(maybeCurrentTrace);
         return sampled
-                ? new SampledDetachedSpan(operation, type, traceId, getParentSpanId(maybeCurrentTrace))
-                : new UnsampledDetachedSpan(traceId);
+                ? new SampledDetachedSpan(operation, type, traceId, parentSpan)
+                : new UnsampledDetachedSpan(traceId, parentSpan);
```

Comment:
Bugfix: parent span information was not propagated through unsampled detached spans.